bundles/org.openhab.persistence.jdbc/src/main/java/org/openhab/persistence/jdbc/db/JdbcTimescaledbDAO.java

```diff
@@ -25,47 +25,33 @@
  * supplements the default settings from JdbcBaseDAO and JdbcPostgresqlDAO.
  *
  * @author Riccardo Nimser-Joseph - Initial contribution
+ * @author Dan Cunningham - Fixes and refactoring
  */
 public class JdbcTimescaledbDAO extends JdbcPostgresqlDAO {
     private final Logger logger = LoggerFactory.getLogger(JdbcTimescaledbDAO.class);
 
-    protected String sqlCreateHypertable;
+    private final String sqlCreateHypertable = "SELECT created from create_hypertable('#tableName#', 'time')";
 
     public JdbcTimescaledbDAO() {
         super();
-
-        initSqlQueries();
     }
```

Comment:
Is it possible to remove this default constructor? It seems to be called using reflection in `JdbcConfiguration.setDBDAOClass`.

Reply:
Yep, constructor can be safely remove now.